.github/workflows/build_packages.yaml

```diff
@@ -551,28 +549,22 @@ jobs:
           EMQX_NAME=${{ steps.pre-meta.outputs.emqx_name }}
         file: source/deploy/docker/Dockerfile
         context: source
+
     - uses: aws-actions/configure-aws-credentials@v1
-      if: github.event_name == 'release' && !github.event.release.prerelease && matrix.profile == 'emqx'
+      if: matrix.profile == 'emqx'
       with:
         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
-    - name: Push image to aws ecr
-      if: github.event_name == 'release' && !github.event.release.prerelease && matrix.profile == 'emqx' && matrix.build_elixir == 'no_elixir'
-      run: |
-        version=${GITHUB_REF##*/}
-        docker pull emqx/emqx:${version#v}
-        docker tag emqx/emqx:${version#v} public.ecr.aws/emqx/emqx:${version#v}
-        aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
-        docker push public.ecr.aws/emqx/emqx:${version#v}
-    - name: Push image to aws ecr (elixir)
-      if: github.event_name == 'release' && !github.event.release.prerelease && matrix.profile == 'emqx' && matrix.build_elixir == 'with_elixir'
+    - name: Docker push to aws ecr
+      if: matrix.profile == 'emqx'
```

Comment:
why the if guard of 'release/tag' is removed? Was that for testing for now?

Reply:
In my opinion, there were too many judgments in the if, which made it hard to read, and whether or not to login did not affect the following steps, so I changed the if

Reply:
But this is the `push` one not the `login one`?

Reply:
Oh, my bad, that was a huge mistake, thanks